crossing-repair.lic

```diff
@@ -12,7 +12,7 @@ class CrossingRepair
   def initialize
     setup
 
-    ensure_copper_on_hand(@repair_withdrawal_amount, @bank_info)
```

Comment:
The rest of crossing-repair uses hometown, so I think this call should also. We are (or are almost?) at the point we could just rename this to repair.lic.